app/frontend/app/controllers/anime/quotes.js

```diff
@@ -37,6 +30,7 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
           });
 
       quote.save();
+      this.get('content').addObject(quote);
```

Comment:
hmm.  I don't know enough to say one way or the other, so I'll just assume you're right